drivers/i2c/dw_i2c_master.c

```diff
@@ -1,4 +1,5 @@
 #include <target/i2c.h>
+#include <target/delay.h>
 #include <driver/dw_i2c.h>
```

Comment:
Note: driver/dw_i2c.hshouldn‘t be directly included by a .c file.
It should be included by driver/i2c.h.
So you can just include <target/i2c.h>